javascript/node/selenium-webdriver/lib/webdriver.js

```diff
@@ -1229,6 +1229,14 @@ class WebDriver {
     if (target && cdpTargets.indexOf(target.toLowerCase()) === -1) {
       throw new error.InvalidArgumentError('invalid target value')
     }
+
+    if (debuggerAddress.match(/\/se\/cdp/)) {
+      if (debuggerAddress.match("ws:\/\/", "http:\/\/")) {
```

Comment:
What would happen here if there is a `wss`?

Reply:
Thank you for bringing that to notice. Have added an additional check for the same and tested it locally with the Grid.